std/src/main/scala/algebra/std/double.scala

```diff
@@ -0,0 +1,76 @@
+package algebra
+package std
+
+import algebra.lattice.Lattice
+import algebra.number.{IsReal, NRoot, Signed}
+import algebra.ring.Field
+import algebra.std.util.StaticMethods
+
+import java.lang.Double.{ longBitsToDouble, doubleToLongBits }
+import java.lang.Long.{ numberOfTrailingZeros, numberOfLeadingZeros }
+import java.lang.Math
+
+import scala.annotation.tailrec
+
+trait DoubleInstances {
+  implicit val doubleAlgebra = new DoubleAlgebra
+
+  val DoubleMinMaxLattice = new Lattice[Double] {
+    def join(x: Double, y: Double): Double = if (x > y) x else y
+    def meet(x: Double, y: Double): Double = if (x < y) x else y
+  }
+}
+
+class DoubleAlgebra extends Field[Double] with NRoot[Double] with Order[Double] with Signed[Double] with IsReal[Double] with Serializable {
```

Comment:
should we comment here that this breaks the laws due to IEEE-754, but that it is approximately a Field?